### PR TITLE
WIP: Signature Help

### PR DESCRIPTION
**Pull request by ctolkien**
(again, PR'ing this early for visibility).

yes.. my tsconfig line endings are borked....

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,68 +1,69 @@
-{
```
Comment (nosami) [inline tsconfig.json]: my eyes

File: lib/omnisharp-atom/features/signature-help.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import ClientManager = require('../../omni-sharp-server/client-manager');
+import OmniSharpAtom = require('../omnisharp-atom');
+import Omni = require('../../omni-sharp-server/omni')
+
+class SignatureHelp {
+
+    constructor(private atomSharper: typeof OmniSharpAtom) {
```
Comment (nosami) [inline lib/omnisharp-atom/features/signature-help.ts]: don't need atomSharper here. You can just use `OmniSharpAtom.addCommand`

Comment (ctolkien) [inline lib/omnisharp-atom/features/signature-help.ts]: Ah k, I'd seen this used elsewhere. Should also update the Code Action bits as it's the same pattern.

**Comment by david-driscoll** [comment]
Closing because I got this added in later :)

**Pull request closed without merging**